tests/src/JIT/HardwareIntrinsics/X86/Lzcnt/Lzcnt.cs

```diff
@@ -26,11 +27,22 @@ static int Main(string[] args)
                     Console.WriteLine("Intrinsic Lzcnt.LeadingZeroCount is called on non-supported hardware.");
                     Console.WriteLine("Lzcnt.IsSupported " + Lzcnt.IsSupported);
                     Console.WriteLine("Environment.Is64BitProcess " + Environment.Is64BitProcess);
-                    return Fail;
+                    testResult = Fail;
                 }
                 catch (PlatformNotSupportedException)
                 {
-                    testResult = Pass;
+                }
```

Comment:
Why do these tests check Is64BitProcessor?

Reply:
The overload that takes a long is only available on x64 systems, it is not encodable on x86.